### PR TITLE
Update Popper wrapper to use newest version

### PR DESCRIPTION
**Pull request by hapurubokka**
We've been using popper.js through v-tooltip, so we need to move away from v-tooltip and make our own tooltip and popper wrappers to update to their newest versions.

`AppPopper`
-------------
Pass the new `fixed` prop if being used on a fixed element, like an item in the top nav or a cbar item, so the popper doesn't stutter on scroll.

Make sure that the intended trigger/reference for the popper is in the default slot of `AppPopper`.

### CODE REVIEW

File: src/_common/popper/popper.styl [hunk 1]
```diff
@@ -7,92 +7,78 @@ $-transition-time = 200ms
 .v-popover
```
Comment (hworld) [inline src/_common/popper/popper.styl:7]: Let's call this something other than v-popper.

File: src/_common/popper/popper.ts [hunk 2]
```diff
@@ -95,46 +139,179 @@ export default class AppPopper extends Vue {
...
+		// user hovers while the popper is in the process of hiding.
+		if (this.isHiding) {
+			this.hideDone();
+		}
+
+		if (this.isVisible) {
+			return;
+		}
+
+		this.hideTimeout = setTimeout(() => this.onShow(), this.actualDelay.show);
```
[the remark this answers is not in the document]
Comment (hapurubokka) [reply src/_common/popper/popper.ts]: I think we do need a `setTimeout` using the `showDelay` prop, right? I changed it to use a separate `showDelayTimer` that we clear in `onMouseLeave`.

File: src/_common/popper/popper.styl (hunk 1 above)
Comment (hapurubokka) [reply src/_common/popper/popper.styl]: renamed to `popper`, and the wrapper for the actual popper/popup element is changed to `popper-wrapper`.

File: src/_common/tooltip/tooltip.ts [hunk 3]
```diff
@@ -1,35 +1,145 @@
...
+		_inner.appendChild(_content);
+		TooltipElement = _tooltip;
+
+		PopperElement = createPopper(trigger, TooltipElement, getOptions(binding.modifiers));
+
+		document.body.appendChild(_tooltip);
+	};
+
+	const onMouseUp = (trigger: any, event: MouseEvent) => {
+		setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (hapurubokka) [reply src/_common/tooltip/tooltip.ts:98]: We don't actually need to make this async since we're not checking for the 'popped' class anymore.

**Pull request merged**